packages/aws-cdk/lib/api/util/sdk.ts

```diff
@@ -53,6 +57,20 @@ export interface SDKOptions {
    * @default Automatically determine.
    */
   ec2creds?: boolean;
+
+  /**
+   * Change the endpoint addresses of the aws sdk services
+   *
+   * The argument supports both comma separated serviceName=URL pairs,
+   * and JSON structures
+   * E.g.
+   * cdk deploy --endpoints 'cloudformation=http://localhost:4581,s3=http://localhost:4572'
+   * cdk deploy --endpoints '{"cloudformation":"http://localhost:4581","s3":"http://localhost:4572"}'
+   *
+   * @default Standard AWS cloud endpoints
```

Comment:
* what's does standard map to? there are multiple partitions to consider. a `@link` to service endpoints would be helpful as well